### PR TITLE
Rework #156

### PR DESCRIPTION
**Pull request by rustyrussell**
I believe the final result of this is better, and hopefully understandable!  If acked, I will squash into one patch and apply.

### CODE REVIEW

File: 09-features.md [hunk 1]
```diff
@@ -5,20 +5,52 @@ They are tracked separately since new flags will likely be added over time.
...
+## Requirements
+
+(Note that the requirements for feature bits which are not defined
+above, can be found in [BOLT #1: The `init` message](#the-init-message)).  The requirements when receiving set bits are defined in the linked section in the table above).
+
+Additional requirements:
+
+* `channels_public`: the sender MUST set exactly one of these bits if
+   it wants to announce the channel publicly, otherwise it MUST set
+   neither.  If it sets one it MUST set the even bit if will fail the
```
Comment (sstone) [inline 09-features.md]: typo: .. even bit if **it** will ...

**Comment by sstone** [comment]
ACK. Much easier to understand indeed !

File: 09-features.md (hunk 1 above)
Comment (rustyrussell) [inline 09-features.md]: I'll push a fixup, though #169 removes this paragraph altogether.

**Comment by rustyrussell** [comment]
@EmelyanenkoK does this seem right to you, too?  You're the one who started this much-needed cleanup!

**Comment by EmelyanenkoK** [comment]
@rustyrussell I find it clear and ready.

**Pull request merged**